how2meet/models.py

```diff
@@ -0,0 +1,45 @@
+
+from sqlalchemy import Column, Integer, String, ForeignKey
+from sqlalchemy.orm import relationship
+from sqlalchemy.types import Date, DateTime
+from .database import Base
+
+
+class Event(Base):
+    __tablename__ = "events"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String(150))
+    date = Column(Date)
+    start_time = Column(DateTime)
+    end_time = Column(DateTime)
+    all_day = Column(bool)
+    locatiom = Column(String(150))
+    host_name = Column(String(50))
+    host_password = Column(String(100))
+    duration = Column(Integer)
+    invites = relationship(
```

Comment:
I would like to better understand this at some point. The tutorial I was looking at used `back_populates` arg and didn't really explain why